lms/services/lti_h.py

```diff
@@ -41,7 +43,7 @@ def sync(self, h_groups, group_info_params):
         :raise HTTPInternalServerError: if we can't sync to h for any reason
         """
 
-        if not self._ai_getter.provisioning_enabled():
+        if not self._application_instance_service.get().provisioning:
```

Comment:
This could have raised `ConsumerKeyError` before and it still can now, no change